invoices_app/models.py

```diff
@@ -16,6 +17,16 @@ def __str__(self):
         return "Status: {}".format(self.name_tax_payer_state)
 
 
+class CompanyUserPermission(models.Model):
+    company = models.ForeignKey(Company, on_delete=models.CASCADE)
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        default=None
+    )
+    permission = None
```

Comment:
What is this "permission" field?

Reply:
This field is for next sprint. We will implement the permission logic than permission will be a foreign key to the permission object